spec/support/vcr.rb

```diff
@@ -0,0 +1,8 @@
+require 'vcr'
+VCR.configure do |c|
+  c.cassette_library_dir = 'spec/cassettes'
+  c.hook_into :webmock
+  c.configure_rspec_metadata!
+  c.default_cassette_options = { record: :new_episodes }
+  c.filter_sensitive_data('<LP_FLOW>') { ENV['LP_FLOW'] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.